modules/fluid_properties/unit/src/Water97FluidPropertiesTest.C

```diff
@@ -677,3 +677,97 @@ TEST_F(Water97FluidPropertiesTest, henry)
   REL_TEST(Kh, _fp->henryConstant(500.0, co2), REL_TOL_CONSISTENCY);
   REL_TEST(dKh_dT_fd, dKh_dT, REL_TOL_DERIVATIVE);
 }
+
+/**
+ * Verify that calculations from conservative variables are accurate
+ */
+TEST_F(Water97FluidPropertiesTest, conservative)
+{
+  auto run_tests = [this](const auto & example)
+  {
+    typedef typename std::decay<decltype(example)>::type TestType;
+    const TestType pressure = 1.01e5;
+    const TestType temperature = 298.15;
+
+    const auto rho = _fp->rho_from_p_T(pressure, temperature);
+    const auto v = 1 / rho;
+    auto e = _fp->e_from_p_T(pressure, temperature);
+
+    auto [p_test, T_test] = _fp->p_T_from_v_e(v, e);
+    REL_TEST(MetaPhysicL::raw_value(pressure), MetaPhysicL::raw_value(p_test), REL_TOL_CONSISTENCY);
```

Comment:
You put all of the `raw_value` in `REL_TEST` and `ABS_TEST`, didn't you?

Reply:
Yes I did do that "later". I can remove the redundancy